exercises/practice/protein-translation/.meta/src/ProteinTranslation.example.elm

```diff
@@ -0,0 +1,104 @@
+module ProteinTranslation exposing (proteins)
+
+import List
+import String
+
+
+proteins : String -> Result String (List String)
+proteins strand =
+    case splitIntoCodons strand of
+        [ "" ] ->
+            Ok []
+
+        codons ->
+            case processCodons [] codons of
```

Comment:
Not sure if it's better, but I think this whole block could be
```elm
codons
 |> processCodons []
 |> Result.map List.reverse
```
and if you could somehow treat the empty case in `processCodons` the whole function could be
```elm
strand
 |> splitIntoCodons
 |> processCodons []
 |> Result.map List.reverse
```

Reply:
Thanks for the review, I'll make the edits tomorrow.